tests/testthat/test-weight_allom.R

```diff
@@ -10,7 +9,7 @@ test_that("weight_allom returns a vector of the same length as the equation tabl
               coords = c(-78, 40),
               new_eqtable = newtab
             )
-            expect_is(weight_test, "numeric")
+            expect_type(weight_test, "double")
```

Comment:
"numeric" is the class; "double" is the type.

``` r
class(1)
#> [1] "numeric"
typeof(1)
#> [1] "double"
```

<sup>Created on 2021-02-13 by the [reprex package](https://reprex.tidyverse.org) (v1.0.0)</sup>